kernel-kit: 1.0.0 release notes (#2)
bottlerocket-os/bottlerocket-kernel-kit: CHANGELOG.md

# 1.0.0 (2024-12-11)

## Build Changes
* Create the new kernel kit from the following core-kit packages: ([#1])
  * grub
  * kernel-5.10
  * kernel-5.15
  * kernel-6.1
  * kmod-5.10-nvidia
  * kmod-5.15-nvidia
  * kmod-6.1-nvidia
  * linux-firmware
  * microcode
  * libkcapi
  * shim
* Update bottlerocket-sdk to v0.50.0

> - [#1](https://github.com/bottlerocket-os/bottlerocket-kernel-kit/pull/1)

Reviewer: ```suggestion
[#1]: https://github.com/bottlerocket-os/bottlerocket-kernel-kit/pull/1
```
a la https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/c57c8e9370d3a69ef57ca08689eae1b927a637d1/CHANGELOG.md?plain=1#L12

# 1.0.0 (2024-12-11)

## Build Changes

> * Creation of a new kernel kit
> * Migration of the below packages from core kit to kernel kit

Reviewer 2: Tense nits:

```suggestion
* Create the new kernel kit from the following core-kit packages ([#1]):
```

  * grub
  * kernel-5.10
  * kernel-5.15
  * kernel-6.1
  * kmod-5.10-nvidia
  * kmod-5.15-nvidia
  * kmod-6.1-nvidia
  * linux-firmware
  * microcode
  * libkcapi
  * shim

> * Update SDK to 0.50.0

Reviewer 2: Consistency with core-kit changelog:

```suggestion
* Update bottlerocket-sdk to v0.50.0
```
